addons/l10n_ar/__manifest__.py

```diff
@@ -1,8 +1,13 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
+<<<<<<< HEAD
     'name': 'Argentina - Accounting',
     'version': "3.1",
+=======
+    'name': 'Argentinian Accounting',
+    'version': "3.2",
+>>>>>>> 24390b24eaa... temp
```

Comment:
```suggestion
    'name': 'Argentina - Accounting',
    'version': "3.2",
```

Hi @jco-odoo 
In order to fix this fw-p only need to do this change.

We only change the version and maintain the name of the module in14.0 because was requested this way in [latam task 350](https://latam-localizations.odoo.com/web#id=350&action=138&model=project.task&view_type=form&menu_id=88)


Best Regards